src/net/headers/content_type.rs

```diff
@@ -0,0 +1,70 @@
+use axum::headers;
+use axum::headers::Header;
+use http::HeaderName;
+use http::HeaderValue;
+
+/// Typed header implementation for the `ContentType` header.
+pub enum ContentType {
```

Comment:
Assuming here `ContentType::Surrealdb` is bincode, Don't we also have a type for revision since a while?

Reply:
I copied these from the Accept, so if its not there it wasn't supported before